indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -101,6 +101,16 @@
                    batch)]
     (es/prepare-batch context (filter-expired-concepts batch) options)))
 
+(defmethod prepare-batch :service
+  [context batch options]
+  ;; Get the service associations as well.
```

Comment:
I thought we said we aren't indexing service associations when indexing a service?

Reply:
/me nods ...

So it looks like here that would just be a straight-up call to `es/prepare-batch` ... 

/me tries that out.

Reply:
Yup, that works; all changes now pushed.